### ISSUE TITLE
Just finished the current version and I have some pointers

### ISSUE THREAD

**Issue by JohnWJarrett** [SaphieNyako/Feywild#115]
### Describe your idea

Great so far, love the wood and the Pixies.

Couple points and possible bugs though:
1: Summer quest line asks for you to kill a Zombie Brute when it should be an Iron golem
2: Winter Pixie's ability trashes a base pretty quickly, more of an annoyance than a problem, I fixed it by building a box with some landscaping for her to live in and summon as many snow golems as her heart desires
3: other than the Spring trees, no other tree seems to drop anything other than wood and leaves, no fey dust
4: I search for a while and never found a single ore block. Got mine from trades

Other than those points, it was really fun what's there so far and can't wait to see what's next.
Also, thanks for the quick updates to my other two reports.

**Comment by SaphieNyako** [comment]
Thank you so much for your support and reports! 

1. Changed the text to "Iron Golem" for next update. This was a leftover from the modpack.
2. In one of the next patches I have a pixie update planned.
Wich will make this ability her Trickery ability, and her main ability will be casting Frost Walk on the player. 
3. Added all loot tables for the leaves. Left the loot tables for the original leaves... Thank you!
4. The ore only spawns in the feywild dimension. I made a note on that in the Guide Book for next update. 

Glad you enjoyed it and Thank you again! 
![2023-03-18_15 57 21](https://user-images.githubusercontent.com/77398437/226114167-49a56fde-ad8c-4873-8b78-0a7e6591e99b.png)
![2023-03-18_16 06 51](https://user-images.githubusercontent.com/77398437/226114187-42aada88-6799-46bc-bf0c-f825e2197949.png)
